model.rb

```diff
@@ -12,7 +12,15 @@
 Sequel::Model.plugin :singular_table_names
 Sequel::Model.plugin :subclasses unless ENV["RACK_ENV"] == "development"
 Sequel::Model.plugin :column_encryption do |enc|
-  enc.key 0, Config.clover_column_encryption_key
+  key = Config.clover_column_encryption_key
+  unless Config.kms_encrypt_clover_column_encryption_key_region.nil?
+    require 'aws-sdk-kms'
+    kms_client = Aws::KMS::Client.new(region: Config.kms_encrypt_clover_column_encryption_key_region)
```

Comment:
Looking at `Aws::KMS::Client` documentation (https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/KMS/Client.html#initialize-instance_method), this seems to be relying on `:credentials` being passed implicitly via environment variables, files, or `Aws.config`. However, this passes `:region` explicitly (even though `:region` can also be passed implicitly). Is there a reason for this difference?

If the credentials are passed via environment variables, then it seems this wouldn't actually fix the issue, only require an extra step, since an attacker with access to the environment variables would be able to call `Aws::KMS::Client` with the environment variables to get the decryption key.

Reply:
> However, this passes :region explicitly (even though :region can also be passed implicitly). Is there a reason for this difference?

Yes, we should remove region here